src/render/OpenGL.cpp

```diff
@@ -1597,6 +1605,11 @@ void CHyprOpenGLImpl::renderTextureInternalWithDamage(SP<CTexture> tex, const CB
         shader->setUniformFloat2(SHADER_FULL_SIZE, m_renderData.pMonitor->m_pixelSize.x, m_renderData.pMonitor->m_pixelSize.y);
     }
 
+    if ((usingFinalShader && *PDT == 0) || CRASHING)
+        shader->setUniformFloat2(SHADER_POINTER, g_pPointerManager->position().x, g_pPointerManager->position().y);
```

Comment:
g_pInputManager->getMouseCoordsInternal, plus it would make sense to translate them to the opengl space